web3/utils/normalizers.py

```diff
@@ -58,12 +56,6 @@ def addresses_checksummed(abi_type, data):
         return abi_type, to_checksum_address(data)
 
 
-@implicitly_identity
-def decode_abi_strings(abi_type, data):
-    if abi_type == 'string':
-        return abi_type, codecs.decode(data, 'utf8', 'backslashreplace')
```

Comment:
Im getting various DecodingErrors when using the updated eth_abi package that has the change to return strings as string types.

Likely the string data in these tests are not encodable.

Reply:
Hm, well feel free to copy in a trace if you want another pair of eyes on it.

BTW, there might be other things besides contract arguments/return values that depend on this `decode_abi_strings()` method, so it's not clear it should be deleted.

Reply:
@carver I moved this to another PR since it needs it own discussion: https://github.com/ethereum/web3.py/pull/974